README.rst

```diff
@@ -563,14 +591,98 @@ saved on ``2020-08-08`` as follows.
 
 .. code-block:: text
 
-    GET /api/v1/topology/{id}/history/?date=2020-08-08
+    GET /api/v1/topology/topology/{id}/history/?date=2020-08-08
 
 Send topology data
 ##################
 
 .. code-block:: text
 
-    POST /api/v1/topology/{id}/receive/
+    POST /api/v1/topology/topology/{id}/receive/
+
+List links
+##########
+
+.. code-block:: text
+
+    GET /api/v1/topology/link/
+
+Create link
+###########
+
+.. code-block:: text
+
+    POST /api/v1/topology/link/
```

Comment:
Yes, It must be a valid JSON

Reply:
Oops, my bad, I didn't explain properly, if I give it those values, it doesn't follow the expected behavior and I get the following:
![ss1](https://user-images.githubusercontent.com/24985760/130071264-548c6d52-3580-4861-8677-6489a9508cd6.png)

Reply:
@atb00ker what we are doing here is not write I think, because, in the api it expects a value in json format, and i guess you passed `0` by removing the empty dict. so the error is raised by `full_clean` method,

But, in the admin panel it allows? because by default the data which is sent is sent as `{"": 0}`, see this is a valid json. Try sending the 0 or 0, 0 as :- {"": 0}, {"": "0,0"} this will work.

If I override this behaviour, then it will not respect JSON format.

Reply:
Got your point. working on it

Reply:
Done